src/misc/SubscriptionDialogs.js

```diff
@@ -62,20 +62,21 @@ export function checkPremiumSubscription(included: boolean): Promise<boolean> {
 	})
 }
 
-export function showMoreStorageNeededOrderDialog(loginController: LoginController,
-                                                 messageIdOrMessageFunction: TranslationKey | lazy<string>
-): Promise<void> {
-	return Dialog.confirm(messageIdOrMessageFunction, "upgrade_action").then((confirm) => {
-		if (confirm) {
-			if (loginController.getUserController().isPremiumAccount()) {
-				import("../subscription/StorageCapacityOptionsDialog").then((StorageCapacityOptionsDialog) => {
-					StorageCapacityOptionsDialog.show()
-				})
-			} else {
-				showNotAvailableForFreeDialog(false)
+/**
+ * @pre isGlobalAdmins is true
```

Comment:
I would just check and throw tbh

Reply:
Currently, we already check in every place that the function is used. But sure, we can check again in case someone uses it in a different place without checking the precondition.